documentation/objects/objects.md

```diff
@@ -17,49 +17,45 @@ weight: 1
 
 As the name suggests, the object classification workflow aims to classify full *objects*, based on object-level features and user annotations.
 An *object* in this context is a set of pixels that belong to the same instance.
-In order to do so, the workflow needs *segmentation* images besides the usual raw image data, that can e.g. be generated with the [Pixel Classification Workflow].
-Depending on the availability of these segmentation images, the user can choose between three flavors of object classification workflow, which differ by their input data:
+Object classification requires a second input besides the usual raw image data: an image that indicates for pixels whether they belong to an object or not, i.e. object predictions or a binary segmentation.
```

Comment:
```suggestion
Object classification requires a second input besides the usual raw image data: an image that indicates for pixels whether they belong to an object or not, i.e. _pixel predictions_, a _binary segmentation image_ or a _label image_.
```
the term "object predictions" could be confused with the output "object predictions" the workflow produces